pkg/blob/nodeserver.go

```diff
@@ -213,13 +225,38 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 
 	klog.V(2).Infof("target %v\nprotocol %v\n\nvolumeId %v\ncontext %v\nmountflags %v\nmountOptions %v\nargs %v\nserverAddress %v",
 		targetPath, protocol, volumeID, attrib, mountFlags, mountOptions, args, serverAddress)
-	cmd := exec.Command("blobfuse", strings.Split(args, " ")...)
 
-	cmd.Env = append(os.Environ(), "AZURE_STORAGE_ACCOUNT="+accountName)
-	cmd.Env = append(cmd.Env, "AZURE_STORAGE_BLOB_ENDPOINT="+serverAddress)
-	cmd.Env = append(cmd.Env, authEnv...)
+	transportOption := grpc.WithInsecure()
+	useBlobfuseProxy := true
+	var output []byte
+	klog.V(2).Infof("NodeStageVolume: sending GRPC call to blobfuseproxy")
+	cc1, err := grpc.Dial("unix://"+d.blobfuseProxyEndpoint, transportOption)
```

Comment:
what's `blobfuseProxyEndpoint` expected prefix? I see in flag, it's starting with `unix://` by default.

Reply:
seems it goes to use proxy by default, there is no error in L233:
```
I0218 19:08:55.138727    8936 nodeserver.go:232] NodeStageVolume: sending GRPC call to blobfuseproxy
I0218 19:08:55.139073    8936 nodeserver.go:251] NodeStageVolume: blobfuseproxy returned.
E0218 19:08:55.139104    8936 nodeserver.go:262] Mount failed with error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial unix /var/lib/kubelet/plugins/blobfuseproxy.sock: connect: no such file or directory", output: 
E0218 19:08:55.139244    8936 utils.go:119] GRPC error: Mount failed with error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial unix /var/lib/kubelet/plugins/blobfuseproxy.sock: connect: no such file or directory", output: 
Mount failed with error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial unix /var/lib/kubelet/plugins/blobfuseproxy.sock: connect: no such file or directory", output: 
```

https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_blob-csi-driver/349/pull-blob-csi-driver-integration/1362478652712816640/build-log.txt